citi-onboarding/Aqua-Jr
# hotfix/expertise-call-to-blog-merge

## What I did
- Deleted unnecessary folder
- Fixed the bug where the button would not pass the slide
- Fixed the bug where the website had a horizontal scrow
- Fixed all hovers

## How to test
1. Access the branch through your terminal:
```
   git checkout develop
   git pull
   git checkout hotfix/expertise-call-to-blog-merge
```
2. Access the server folder and run the server:
```
   cd /server
   nodemon index.js
```

3. Access the client folder in another terminal window and start React:
```
   cd /client
   yarn start
```
4. In your browser, go to:
``` 
   localhost:3000
```
5. Check if it is responsive (from 300px to 3000px)
6. Check if something is different from the mockup in both versions
7. Check if the arrows are passing the slide normally
8. Check if the page has horizontal scrow
9. Check if the arrow have hovers
